tensorboard/uploader/BUILD

```diff
@@ -145,6 +144,18 @@ py_test(
     ],
 )
 
+py_test(
+    name = "uploader_subcommand_test",
```

Comment:
nit: Could you move this build rule to just under `uploader_subcommand`, please?

Reply:
Oops, thanks :)